docs/UserGuide.md

```diff
@@ -213,10 +214,25 @@ Examples:
 
 ### Clearing all entries : `clear`
 
-Clears the current list of entries from EmployeeManager.
+Clears the current displayed list of entries from EmployeeManager.
 
 Format: `clear`
 
+### View command history : `history`
+
+Displays up to the last `N` valid user commands that have been executed, starting from the previous command. 
+If the number of valid user commands is less than `N`, it shows all available commands.
```

Comment:
Would it be clearer to write "it shows all valid user commands that have been executed" because "it shows all available commands" could be interpreted as showing all commands that a user can execute?